packages/@sanity/base/src/components/ElementContainerQuery/ElementContainerQuery.tsx

```diff
@@ -0,0 +1,28 @@
+import {useElementRect} from '@sanity/ui'
```

Comment:
Adding this here, this component was a creation of @hermanwikner and I'm so thankful 🙌